Add pictorial keyboard bindings

clockbrain opened the pull request:
Add a page that shows the keyboard controls on an image of an ANSI 104 keyboard.
Note: based on layout data from https://www.keyboard-layout-editor.com/ but doesn't support different layouts, just the ANSI 104.

- Added and fixed some missing control categories.
- Changed unused social category to camera.
- Moved all camera controls to camera category
- Softened some colors and changed some blacks to different colors
- Re-enabled color by category option (unsure why this was disabled)
- Added settlement controls
- Removed duplicate controls: YawAxisRaw, RollAxisRaw, PitchAxisRaw
- Renamed HyperSuperCombination to 'Frame Shift' as that is what is it labelled as in the game

Example of keyboard image produced:
![Sample keyboard binds](https://github.com/user-attachments/assets/f5350078-4d28-4aec-92fa-821d693a4808)

clockbrain committed:
Initial working keyboard layout

clockbrain committed:
Fix showing modifier keys.
Separate settlement group.

clockbrain committed:
Tidy key position passing and refactor legend code

clockbrain committed:
Bump version

brammmers commented:
Very nice addition! Thank you.

How difficult would it be to render an ISO layout type map as well?

clockbrain commented:
Do you mean like an ISO 105 from the preset options on https://www.keyboard-layout-editor.com/ ?
Shouldn't be too hard but would also have to consider adding user selection of keyboard layouts to the front page. Leave it with me to explore.

clockbrain committed:
Add ISO 105 keyboard and option to allow selection.
Also add Cyrillic keyboard as a test of exotic keys found in bind files dulclq & jbqvjt.
Keys not present on keyboard are listed at bottom of page

clockbrain commented:
I think it works reasonable well now but I've found a few keys in bind files which I don't map to these keyboards. For example Key_Asterisk which is a shifted key but E-D treats shift as a modifier so it isn't possible to bind asterisk to a control on my keyboard. I can only assume these are produced with extended or international keyboards. Too many varieties to cater for at the moment but the code can be expanded to other keyboards in future.

clockbrain commented:
In case I wasn't clear from my last commit note it does now have the option of showing different keyboard layouts:
<img width="340" height="134" alt="image" src="https://github.com/user-attachments/assets/ee9e0c94-b7fa-4b13-b35f-8158ab9b4712" />


The **ISO 105** keyboard: 
<img width="1501" height="798" alt="image" src="https://github.com/user-attachments/assets/19ef813e-cfbf-454e-8cd3-4284bfe6be44" />

I also added the **Cyrillic** keyboard option because I found a couple of bind files using those keys (e.g. dulclq & jbqvjt) and wanted to make sure the keyboard generation code was as tested as possible:
<img width="1724" height="930" alt="image" src="https://github.com/user-attachments/assets/6b6ce985-0ac6-481c-a023-0925357ae1a8" />

I don't plan to do any more changes to this keyboard generation code unless any bugs are reported.

brammmers commented:
Thanks for taking the time to do this. I'll try and find the time to merege this into the main branch.
